hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/rate/Releasable.java

```diff
@@ -0,0 +1,5 @@
+package pl.allegro.tech.hermes.consumers.consumer.rate;
+
+public interface Releasable {
```

Comment:
Well I thought that combination of type with variable name makes it readable but can't argue if it isn't in the end. I will introduce something more meaningful.